static/js/components/NewShift.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import Form from "@rjsf/material-ui";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import { showNotification } from "baselayer/components/Notifications";
+import { submitShift } from "../ducks/shift";
+import { fetchShifts } from "../ducks/shifts";
+
+const NewShift = () => {
+  const groups = useSelector((state) => state.groups.userAccessible);
+
+  const dispatch = useDispatch();
+
+  if (!groups) {
+    return <CircularProgress />;
+  }
+
+  const handleSubmit = async ({ formData }) => {
+    const result = await dispatch(submitShift(formData));
+    if (result.status === "success") {
+      dispatch(showNotification("Shift saved"));
+      dispatch(fetchShifts());
+    }
+  };
+
```

Comment:
show the current UTC to help the user filling this out?

Reply:
Added